### PR TITLE
Run as non root

### PR DESCRIPTION
**Pull request by bdwyer2**
If this PR is okay I will apply the same to all MAX Models

### CODE REVIEW

**Comment by bdwyer2** [comment]
max-base PR: https://github.com/IBM/MAX-Base/pull/42

**Comment by bdwyer2** [comment]
Tests failed due to an unrelated bandit bug

File: Dockerfile [hunk 1]
```diff
@@ -29,4 +28,6 @@ RUN if [ "$use_pre_trained_model" = "true" ] ; then \
 
 EXPOSE 5000
 
-CMD python /workspace/app.py
+USER max
```
Comment (xuhdev) [inline Dockerfile:31]: Do we want to repeat this line here?

Comment (bdwyer2) [reply Dockerfile:31]: Just tested it out - you are correct. I'll remove this line

**Review by xuhdev — APPROVED** [review]

**Pull request merged**